docs/references/global-shell.md

```diff
@@ -0,0 +1,153 @@
+---
+id: global-shell
+title: Global Shell
+---
+
+The Global App Shell is a new feature in DHIS2 v42. It’s a shared launching point for all of the apps: when you visit DHIS2, the Global Shell will open, and then the Global Shell will load apps inside of itself.
+
+Its intention is to create a consistent experience across DHIS2, consolidate functionality that’s needed across all the apps, and make it easier to roll out shared features to all the apps at once. The first examples of these new features include a redesign of the DHIS2 header bar and the introduction of a new navigation menu, called the Command Palette.
+
+![The Command Palette](./assets/global-shell-command-palette.png)
+
+### Incremental design
+
+The Global Shell is currently designed to require as few changes from apps as possible. In later stages of its implementation, it will require some more updates from apps, but will hoist a number of features that are common to all apps up to itself, improving performance and simplifying the apps. These features might include fetching common requests, managing alerts and localization, and a more semantic connection for routing and PWA communication.
```

Comment:
is "hoist" a well-understood word .. I understand from JS context, but takes me a bit of thinking to realise what it means generally

Reply:
Hmm good question -- in general speech it is well recognized to mean "lift up", but I realize I'm going for kinda the same meaning as "[hoisting state](https://reactpatterns.js.org/docs/state-hoisting/)" in React... I'll do a little more research and see if this terminology is common

Reply:
After some more research, it seems like you're right: this isn't a common way to use it -- I'll switch it to something else! Thanks for pointing that out 😀